Nooriyah lohani (#30)
USRSE/rse-stories: _posts/2020-11-26-nooriyah-lohani.md

---
layout: post
title: "Equal Footing"
author: "@pweschmidt"
rse: "Nooriyah Lohani"
phenotype: 
excerpt: "Let's put RSEs on equal footing with academics and professionals and get them recognised and formal roles, says Nooriyah Lohani from NESI in New Zealand. That requires to reach decision makers. And for that you need to build and foster a strong community first. "

> date: 2020-11-26 8:30:00

Reviewer: ```suggestion
date: 2020-12-03 8:30:00
```

external_media: https://us-rse.org/rse-stories-episodes-1/2020/rse-stories-nooriyah-lohani-episode-44.mp3
length: 12705877
duration: "00:26:27"
explicit: "no"
resources:
  - name: New Zealand eScience Infrastructure
    url: https://www.nesi.org.nz
  - name: Mystery Genes
    url: https://twitter.com/mysterygenes
  - name: NeSI twitter
    url: https://twitter.com/nesi_nz 
--- 

> In some countries research software engineers have established roles and there are (some) career paths available to them. In New Zealand, however, there is still a way to go. Meet [Nooriyah Lohani](https://twitter.com/mysterygenes), research community advisor at [NESI](https://www.nesi.org.nz). Nooriyah's background is in bio informatics and she recently submitted a paper on forecasting the effect of influenza like viruses on the health system in New Zealand.

Reviewer: ```suggestion
In some countries research software engineers have established roles and there are (some) career paths available to them. In New Zealand, however, there is still a way to go. Meet [Nooriyah Lohani](https://twitter.com/mysterygenes), research community advisor at [NESI](https://www.nesi.org.nz). Nooriyah's background is in bioinformatics and she recently submitted a paper on forecasting the effect of influenza like viruses on the health system in New Zealand.
```

Buddying up with Australia, Nooriyah and colleagues of hers in New Zealand are building a community of RSEs in their countries. Their first big milestone was an Australia/new Zealand RSE conference - entirely virtual in Covid times - with ca 150 attendees. 

> Nooriyah wishes to see RSEs on equal footing with academics and other professionals. Ultimately, the success of achieving this goal hinges on working with decision makers. As Nooriyah says: "we need to reach the decision makers. But to do that we need a strong community backing."

Reviewer: ```suggestion
Nooriyah wishes to see RSEs on equal footing with academics and other professionals. Ultimately, the success of achieving this goal hinges on working with decision makers. As Nooriyah says, "We need to reach the decision makers. But to do that we need a strong community backing."
```
